docs/install_linux_mac.rst

```diff
@@ -170,13 +170,14 @@ one-by-one:
 
 .. code-block:: none
 
-    brew install python --with-brewed-openssl
+    brew install python@3.8
     brew install git
-    brew tap caskroom/versions
-    brew cask install homebrew/cask-versions/adoptopenjdk11
+    brew cask install AdoptOpenJDK/openjdk/adoptopenjdk11
 
-It's possible you will have network issues. If so, go in your Applications folder, inside it, go in
-the Python 3.8 folder then double click ``Install certificates.command``.
+.. attention::
```

Comment:
If they install through brew, it is installed with —-with-openssl with an openssl package that was built for brew, so there isn’t a need for the Install certificates.command note - they will only have that file if they have installed through the Python installer application.